Template/Variable/MatomoConfigurationVariable.php

```diff
@@ -141,10 +163,22 @@ public function getParameters()
                 $field->description = Piwik::translate('TagManager_MatomoConfigurationMatomoCustomCookieTimeOutEnableDescription');
                 $field->condition = '!disableCookies';
             }),
-            $this->makeSetting('customCookieTimeOut', '393', FieldConfig::TYPE_INT, function (FieldConfig $field) {
```

Comment:
@EPNW-Eric will it be okay if we do not change the name here ?

Reply:
Hm I think we could keep the name `customCookieTimeOut`, but in my opinion it would be cleared if its changed to `visitorCookieTimeOut` since it's just the timeout for the visitor cookie, but not for the other two types of cookies (session and referral). Also, `visitorCookieTimeOut` would be consistent with the new `sessionCookieTimeOut` and `referralCookieTimeOut`. Is the reason behind keeping it migration?

Reply:
@EPNW-Eric Yes the reason is to keep the migration simple, you can adjust the description to mention that it will affect the visitor cookie timeout only